packages/desktop-client/src/components/budget/SidebarGroup.tsx

```diff
@@ -122,6 +126,12 @@ export function SidebarGroup({
                     onDelete(group.id);
                   } else if (type === 'toggle-visibility') {
                     onSave({ ...group, hidden: !group.hidden });
+                  } else if (type === 'apply-multiple-category-template') {
```

Comment:
Do you really need to mess with the categories both here and the template file?  It would be nice if you could stick to just one or the other.

Reply:
The other goaltemplate function also work only with categoryIds so thats why i filter them before. I can also send the whole list of categories and use that if you  think thats cleaner

Reply:
Its fine as is, but if the two side could be consolidated as much as possible to not need the extra steps on both sides, that would be prefered.  Could you pass the category group from here and not filter to categories?